src/components/StatusPanel.vue

```diff
@@ -22,6 +27,21 @@ export default {
       this.$nextTick(() => {
         terminal.scrollTop = terminal.scrollHeight;
       });
+    },
+    /**
+     * toggleTerminal keymap. Attempts to load the keymap data from
+     * a predefined known file path.
+     * @param {boolean} isAutoInit If the method is called by the code
+     * @return {object} promise when it has completed
+     */
+    toggleTerminal() {
+      if (document.getElementById('terminal').getAttribute('class')) {
```

Comment:
could you do it the vue way? like: https://vuejs.org/v2/guide/class-and-style.html

Reply:
2aba62ef78e42e1e87323fa0fadc08f8bde78b1a?